tests/expected/function-contract/gcd_failure_code.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: -Zfunction-contracts
+type T = u8;
+
+/// Euclid's algorithm for calculating the GCD of two numbers
```

Comment:
Could you add a link where did you extra it from? This might not be necessary here because it probably comes from the past internship. If that's true, ignore it and keep it as note for future tests.